CODEOWNERS

```diff
@@ -1 +1,16 @@
-/eng/SourceBuild* @dotnet/source-build-internal
\ No newline at end of file
+/eng/SourceBuild* @dotnet/source-build-internal
```

Comment:
Why is github complaining about this? This line is same as in the other repos - https://github.com/dotnet/sdk/blob/7822425c3ea35e0bb46e57fad11663e7cbf9f850/CODEOWNERS#L106

Reply:
```
Unknown owner on line 1: make sure the team @dotnet/source-build-internal exists, is publicly visible, and has write access to the repository
/eng/SourceBuild* @dotnet/source-build-internal
```

Reply:
Has that team been given any rights to this repo? I don't see it in the list.

Reply:
I guess not! Who can fix this?

Reply:
cc @joperezr